### ISSUE TITLE
Vala gives error at compiling Pino

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#173]
```
Vala updated its version from 0.7.0 to 0.8.0.
Since then, I have some errors at compiling Pino.
There's an evidence → http://paste2.org/p/749023
```

Original issue reported on code.google.com by `thirteen...@gmail.com` on 1 Apr 2010 at 3:46

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 1 Apr 2010 at 5:34
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by 9ee6a452fc revision.
```

Original comment by `tro...@gmail.com` on 1 Apr 2010 at 6:18
- Changed state: **Fixed**
